meta/saiserialize.c

```diff
@@ -803,8 +808,20 @@ int sai_serialize_ip_address(
 
         default:
 
-            SAI_META_LOG_WARN("invalid ip address family: %d", ip_address->addr_family);
-            return SAI_SERIALIZE_ERROR;
+            for (idx = 0; idx < sizeof(*ip_address); idx++) {
```

Comment:
put { in new line, add empty lines before each if and after each }

Reply:
ACK